avm/utilities/tools/helper/Set-ModuleFileAndFolderSetup.ps1

```diff
@@ -36,131 +36,137 @@ Results into:
 #>
 function Set-ModuleFileAndFolderSetup {
 
-    [CmdletBinding(SupportsShouldProcess = $true)]
-    param (
-        [Parameter(Mandatory = $true)]
-        [string] $FullModuleFolderPath,
-
-        [Parameter(Mandatory = $false)]
-        [string] $CurrentLevelFolderPath
-    )
-
-    if ([String]::IsNullOrEmpty($CurrentLevelFolderPath)) {
-        # First invocation. Handling provider namespace
-        $resourceTypeIdentifier = ($FullModuleFolderPath -split '[\/|\\]{1}avm[\/|\\]{1}(res|ptn)[\/|\\]{1}')[2] # avm/res/<provider>/<resourceType>
-        $providerNamespace, $resourceType, $childResourceType = $resourceTypeIdentifier -split '[\/|\\]', 3
-        $avmModuleRoot = ($FullModuleFolderPath -split $providerNamespace)[0]
-        $currentLevelFolderPath = Join-Path $avmModuleRoot $providerNamespace $resourceType
+  [CmdletBinding(SupportsShouldProcess = $true)]
```

Comment:
Is the tab size change intended? Asking because it makes the review slightly harder (from phone at least 😜)

Reply:
updated :)

Reply:
Waiting for the environment to stabilize to avoid additional failed pipelines